client/pom.xml

```diff
@@ -45,6 +45,26 @@
             <groupId>com.sun.jersey</groupId>
             <artifactId>jersey-client</artifactId>
         </dependency>
+
+        <!-- CXF still using jakarta.ws.rs-api 2.x, we have here 3.X  e.g. package javax.ws.rs.  -->
```

Comment:
jakarta is the newer version, do you mean javax.ws.rs as the old one?

Reply:
CFF needs jakarta.ws.rs-api 2.X the one that is still with java package javax.ws.rs
jakarta.ws.rs-api 3.0.0 comes with  jersey 3.0.0  and is renamed to  jakarta.ws.rs.  you know why ..
package javax.ws.rs  that is needed also present in classpath of this test project as jsr311 
  But this is  old version of JAX-RS 1.0  that current CXF will not functions.

To make CXF work with this classpath  I added one another  groupId javax.ws.rs, artifactId:javax.ws.rs-api
probably need to add exclusion to   jersey1

```
            <dependency>
                <groupId>com.sun.jersey</groupId>
                <artifactId>jersey-client</artifactId>
                <version>${version.jersey1}</version>
        <exclusions>
            <exclusion>
                <groupId>javax.ws.rs</groupId>
                <artifactId>jsr311-api</artifactId>
            </exclusion>
        </exclusions>
            </dependency>
```

Reply:
@Hakky54   FIY  artifacts of  jakarta.ws.rs-api  v 2.x   are using java packages  javax.ws.rs  they transitioned to jakarta  packages only recently in 3.x versions   for EE 9 ...